swarm/pss/pss.go

```diff
@@ -654,9 +654,9 @@ func (self *Pss) SendSym(symkeyid string, topic Topic, msg []byte) error {
 	psp, ok := self.symKeyPool[symkeyid][topic]
 	self.symKeyPoolMu.Unlock()
 	if !ok {
-		return fmt.Errorf("invalid topic '%s' for symkey '%s'", topic, symkeyid)
+		return fmt.Errorf("invalid topic '%s' for symkey '%s'", topic.String(), symkeyid)
```

Comment:
I think it is enough to use %v instead of %s and then the String method will be automatically invoked by the formatter

Reply:
Unfortunately the String method is not invoked in this case, without changing the underlying code, I think this is a simple solution for an error message.